robotino_bringup/env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export MY_CATKIN_WORKSPACE=~/catkin_ws
+source /etc/cob.bash.bashrc
```

Comment:
right, I forgot this, this is just to change the color of the bash setup and difference the pcs , not needed...